java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerIT.java

```diff
@@ -47,29 +46,41 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 
-@Test(groups = { "encrypted" })
+/**
+ * Tests for verifying the behavior of {@link EncryptedServerSideMultipartManager} with
+ * {@link MantaClient}.
+ */
+@Test(groups = {"encryptable", "multipart"})
```

Comment:
My objection to the word "encryptable" is that the word sounds a bit unnatural. Why is it preferred over "encrypted"?